src/game_overlay.c

```diff
@@ -72,24 +72,22 @@ void LoadOverlayGDB(const FSOverlayID overlayID)
 }
 #endif // GDB_DEBUGGING
 
-static void FreeOverlayAllocation(PMiLoadedOverlay *param0)
+static void FreeOverlayAllocation(PMiLoadedOverlay *loadedOverlays)
 {
-    GF_ASSERT(param0->isActive == 1);
-    GF_ASSERT(FS_UnloadOverlay(MI_PROCESSOR_ARM9, param0->id) == TRUE);
+    GF_ASSERT(loadedOverlays->isActive == 1);
```

Comment:
```suggestion
    GF_ASSERT(loadedOverlays->isActive == TRUE);
```

And similar for other occurrences in this file.

Reply:
I have applied these and the other suggestions in the new commit